test_conformance/relationals/test_comparisons_fp.cpp

```diff
@@ -341,9 +342,25 @@ int RelationalsFPTest::test_equiv_kernel(unsigned int vecSize,
         {
             if (expected[j] != outData[i * vecSize + j])
             {
-                verror_msg(i, j, vecSize, expected[j], outData[i * vecSize + j],
-                           inDataA[i * vecSize + j], inDataB[i * vecSize + j]);
-                return -1;
+                bool acceptFail = true;
+                if (std::is_same<T, cl_half>::value)
+                {
+                    bool in_denorm = IsHalfSubnormal(inDataA[i * vecSize])
+                        || IsHalfSubnormal(inDataB[i * vecSize]);
```

Comment:
Needs to be fixed like this for accessing the correct index:
                    bool in_denorm = IsHalfSubnormal(inDataA[i * vecSize + j])
                        || IsHalfSubnormal(inDataB[i * vecSize + j]);

Reply:
Good catch, thanks!